breaking-front/src/pages/Financial/components/TransactionForm/TransactionForm.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import * as Style from 'pages/Financial/components/TransactionForm/TransactionForm.styles';
+
+const TransactionForm = ({ type, onSubmit }) => {
+  const [amount, setAmount] = useState(0);
+  const [amountInput, setAmountInput] = useState(false);
+
+  const handleChange = (event) => {
+    let value = Number(event.target.value);
+    if (Number.isNaN(value)) return;
+    setAmount(value);
+  };
+
+  const toggleAmountInput = () => {
+    setAmountInput((pre) => !pre);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (amount === 0) return alert('금액을 입력해주세요.');
+
+    onSubmit({ amount, setAmount });
```

Comment:
```suggestion
    onSubmit({ amount });
```

setAmount는 사용되지 않는 것 같던데 맞나요?